.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Steps to Reproduce
+
+1. 
+2. 
+3. 
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
+
+What is the current behavior?
+
+### On which platforms did you notice this
+
+- [ ] MacOS
+- [ ] Linux 
+- [ ] Windows
+
+**Version Used**:
+
+### Backtrace/Stacktrace
```

Comment:
not sure we need to mention both backtrace and stacktrace, they're used pretty much interchangeably anyway.

stacktrace seems to be more commonly known from what I've seen.

Reply:
maybe it's a good place to explain that there are "native backtraces" and "managed stacktraces" and how to obtain them. Depending on the issue, those can be useful.

Reply:
and probably worth mentioning that it is possible to attach files to github issues via drag-n-drop? (Cause it's unclear and there is no "attach" button) 🙂

Reply:
don't you get this: 
![image](https://user-images.githubusercontent.com/1376924/33861614-14d2eef2-deb5-11e7-81a3-3c8a53f87857.png) ?